### PR TITLE
Create ChatMessage.tsx

### PR DESCRIPTION
**Pull request by antdke**
Issue #1 

I created ChatMessage.tsx
I added styling to it from a style object previously owned by ChatMessages.tsx

### CODE REVIEW

File: client/src/components/ChatBox.tsx [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 import {
   resourceToConversation,
   MessageResource,
   resourceToMessage
-} from "../lib/jsonApi";
-import { RailsRequest, Conversation } from "../types";
-import uuidv1 from "uuid/v1";
-import ChatMessages from "./ChatMessages";
-import NewMessageForm from "./NewMessageForm";
+} from '../lib/jsonApi';
```
Comment (NicholasLYang) [inline client/src/components/ChatBox.tsx]: Could we figure out why this change is happening?

File: client/src/components/ChatMessage.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+const messageStyle = {
+  display: 'flex',
+  flexDirection: 'column',
+  width: '200px',
+  alignItems: 'flex-end',
+  padding: '10px',
+  borderRadius: '10px',
+  backgroundColor: '#09f',
+  margin: '10px'
+} as any;
```
Comment (NicholasLYang) [inline client/src/components/ChatMessage.tsx]: Let's please not do that. Using the any type basically defeats the purpose of TypeScript. Could you provide the error and then we can work through it?

File: client/src/components/ChatBox.tsx (hunk 1 above)
Comment (antdke) [reply client/src/components/ChatBox.tsx]: I think I have a tool that automatically makes double quotes into single quotes (for strings). It was recommended during a tutorial

Comment (NicholasLYang) [reply client/src/components/ChatBox.tsx]: Could we deactivate it? We should keep this consistent and prettier already changes everything to double quotes

Comment (antdke) [reply client/src/components/ChatBox.tsx]: Deactivated. And everything seemed to revert back

File: client/src/components/ChatBox.tsx [hunk 3]
```diff
@@ -50,8 +50,10 @@ class ChatBox extends React.Component<Props, State> {
   };
 
   handleNewConversation = (convoPromise: Promise<Response>) => {
-    convoPromise.then(res => res.json())
+    convoPromise
+      .then(res => res.json())
       .then(res => {
+        console.log(res);
```
Comment (NicholasLYang) [inline client/src/components/ChatBox.tsx]: Now that we're not getting this error, let's remove the console.log

Comment (antdke) [reply client/src/components/ChatBox.tsx]: Okay. done 👍
